Documentation/spelling_wordlist.txt

```diff
@@ -333,6 +333,7 @@ decrypted
 deepcopy
 deletetopic
 demux
+dependant
```

Comment:
I think the correct spelling is `dependent`, and it seems unused in your commit anyway. Can you please remove it?